development/.env

```diff
@@ -1,2 +1,2 @@
 DOCKER_ORG=usdotfhwastoldev
-DOCKER_TAG=develop
\ No newline at end of file
+DOCKER_TAG=develop
```

Comment:
This should be develop-development

Reply:
This tag doesn't represent the carma-config's tag, but the other images' tags such as carma-platform, carma-base etc which all just has `develop`. The `develop-development` tag is handled by `build-image.sh` script which still makes correct tags out of the folder names.